frontend/src/hocs/withScreencast/components/VideoRecordedDialog/VideoRecordedDialog.tsx

```diff
@@ -60,6 +60,10 @@ const VideoRecordedDialog: FunctionComponent<Props> = ({
         Dialog: {
           style: {
             width: "80vw",
+            borderTopLeftRadius: "0.25rem",
```

Comment:
Same comment as above, we might actually not need to set this here since it is already taken care on in the shared Model implementation.

Reply:
You're right, removed 👍